app/main.py

```
@@ -1 +1,47 @@
-# write your code here
+def copy_file(command: str) -> None:
+    """
+    Copies a file in the current directory like the Linux cp command.
+
+    Args:
+        command: A string with the command "cp", the file name to copy,
+                 and the new file name, separated by spaces.
+    """
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return  # Invalid command
+
+    source_file = parts[1]
+    destination_file = parts[2]
+
+    if source_file == destination_file:
+        return  # Do nothing if source and destination are the same
+
+    try:
+        with open(source_file, "r") as file_in, \
+                open(destination_file, "w") as file_out:
+            file_out.write(file_in.read())
+    except FileNotFoundError:
+        pass  # do nothing if file not found.
```

The `except FileNotFoundError` block currently does nothing when a file is not found. It would be more user-friendly to provide a message indicating that the source file was not found, rather than silently passing.